src/components/events/LiveTaskAttendeePopup.tsx

```
@@ -0,0 +1,254 @@
+  };
+
+  const handleDismiss = () => {
+    setDismissedTaskIds((prev) => {
+      const next = new Set(prev);
+      next.add(activeTask!.id);
+      return next;
+    });
+  };
+
+  return (
+    <div role="dialog" aria-modal="true" aria-label="Micro-task available"
```

ESLint

[error] Replace `·role="dialog"·aria-modal="true"` with `⏎······role="dialog"⏎······aria-modal="true"⏎·····`

(prettier/prettier)